### ISSUE TITLE
"event is not defined" error in browser console in Firefox

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3355]
### Expected behavior
No errors

### Actual behavior
Receive error in console
```
ReferenceError: event is not defined
[Learn More]
productGridItemsContainer.js:52:8
ProductGridItemsContainer/_this.onPageClick
productGridItemsContainer.js:52:8
<anonymous>
```

### Steps to reproduce the behavior

1. Launch Reaction with FF

### Versions
```
Node: 8.9.1
NPM: 5.5.1
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.23.1
Reaction: 1.5.11
Reaction branch: master
Docker: 17.09.0-ce
```

**Comment by codejockie** [comment]
Fixed.

**Comment by spencern** [comment]
Resolved in release [v1.6.4](https://github.com/reactioncommerce/reaction/releases/tag/v1.6.4)
